packages/samples/test/output/visibility/openapi.yaml

```diff
@@ -211,9 +211,6 @@ components:
           $ref: '#/components/schemas/PersonUpdateItem'
         relationship:
           type: string
-      required:
-        - person
-        - relationship
```

Comment:
Hmm.... I don't think this is intended.

Reply:
I know what happened here. The Item flag should overrule the Patch flag because when you patch an array you have to replace the whole array and so the items inside the array have to be complete with a full set of required properties. Before the item flag did this because we checked visibility is Update and only update with no flags to make optional. 

(I haven't reviewed the whole PR yet.)

Reply:
Perfect. I'll write a test case for this

Reply:
Okay, fixed.